app/assets/javascripts/setup.js

```diff
@@ -0,0 +1,6 @@
+$('.btn-group-toggle').click(function() {
+  $btnGroup = $(this);
```

Comment:
Forgot `var` keyword before `$btnGroup`. This only works because we are not in `use strict` (recommended btw) mode. 

After 1.0 we will add eslint and move to ES6 (hopefully) and these catches won't need manual review.